clients/unshaded/src/main/java/tachyon/client/table/TachyonRawTablesCore.java

```diff
@@ -52,17 +68,40 @@ SimpleRawTable create(TachyonURI path, int numColumns, ByteBuffer metadata)
    * @throws IOException if a non Tachyon related I/O error occurs
    * @throws TachyonException if an internal Tachyon error occurs
    */
-  RawTableInfo getInfo(SimpleRawTable rawTable) throws IOException, TachyonException;
+  RawTableInfo getInfo(RawTable rawTable) throws IOException, TachyonException;
+
+  /**
+   * Gets the number of partitions currently in the {@link RawColumn}. Each partition is a
+   * separate {@link TachyonFile}
+   *
+   * @param column the raw column containing the partitions
+   * @return the number of partitions currently in the column
+   * @throws IOException if a non Tachyon related I/O error occurs
+   * @throws TachyonException if an internal Tachyon error occurs
+   */
+  int getPartitionCount(RawColumn column) throws IOException, TachyonException;
 
   /**
    * Gets a handler for the given raw table, if it exists.
    *
    * @param path the path of the table in Tachyon space
-   * @return a {@link SimpleRawTable} representing the table
+   * @return a {@link RawTable} representing the table
+   * @throws IOException if a non Tachyon related I/O error occurs
+   * @throws TachyonException if an internal Tachyon error occurs
+   */
+  RawTable open(TachyonURI path) throws IOException, TachyonException;
+
+  /**
+   * Gets the file handler for a partition of a column. A partition should be accessed through
```

Comment:
I think we are using handler/handle to refer to the same concept, I've created a JIRA ticket to address this for other portions of the codebase: https://tachyon.atlassian.net/browse/TACHYON-1317